### PR TITLE
Feature/business login when creating plan

### PR DESCRIPTION
**Pull request by Kyubinhan**

### CODE REVIEW

File: server/libs/db2/model/plan.js [hunk 1]
```diff
@@ -59,6 +59,13 @@ export default class Plan extends Model {
     return 'plan';
   }
 
+  static async findLatestWithStatusExtension(db, where) {
+    // get the latest plan by giving offset and limit
+    const order = ['id', 'desc'];
+    const plan = await this.findWithStatusExtension(db, where, order, 1, 1);
```
Comment (jleach) [inline server/libs/db2/model/plan.js]: 1, 1, reads like magic numbers. Can you define them above like `const fooBar = 1` and `const bamBla = 1`. Then pass those in as params.

File: server/router/routes/plan.js [hunk 2]
```diff
@@ -120,6 +120,22 @@ router.post('/', asyncMiddleware(async (req, res) => {
       }
     }
 
+    /*
```
Comment (jleach) [inline server/router/routes/plan.js]: Can we delete dead code?

Comment (Kyubinhan) [reply server/router/routes/plan.js]: This code block is what I just added. It is temporarily commented out.

File: server/libs/db2/model/plan.js (hunk 1 above)
Comment (Kyubinhan) [reply server/libs/db2/model/plan.js]: for sure!

**Pull request merged**